lesson_19/page_objects/register_page.py

```diff
@@ -0,0 +1,96 @@
+import random
+
+from selenium.webdriver.common.by import By
+
+from ..page_objects.base_page import BasePage
+from ..page_objects.register_result_page import RegisterResultPage
+
+
+class RegisterPage(BasePage):
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    __gender_male_checkbox = (By.XPATH, "//input[@id='gender-male']")
+    __gender_female_checkbox = (By.XPATH, "//input[@id='gender-female']")
+    __first_name_field = (By.XPATH, "//input[@id='FirstName']")
+    __last_name_field = (By.XPATH, "//input[@id='LastName']")
+    __email_field = (By.XPATH, "//input[@id='Email']")
+    __password_field = (By.XPATH, "//input[@id='Password']")
+    __confirm_password_field = (By.XPATH, "//input[@id='ConfirmPassword']")
+    __register_button = (By.XPATH, "//input[@id='register-button']")
+    __first_name_error = (By.XPATH, "//span[contains(text(), 'First name')]")
+    __last_name_error = (By.XPATH, "//span[contains(text(), 'Last name')]")
+    __email_error = (By.XPATH, "//span[@for='Email']")
+    __password_error = (By.XPATH, "//span[@for='Password']")
+    __repeat_password_error = (By.XPATH, "//span[@for='ConfirmPassword']")
+
+    def sign_gender_checkbox(self, gender):
+        if gender == 'male':
+            element = self._wait_element(self.__gender_male_checkbox)
+            element.click()
+        else:
```

Comment:
elif because if I pass some incorrect data it will set female

Reply:
It is fixed